sdk/core/azure-core/src/main/java/com/azure/core/http/MatchConditions.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@mssfang Would this make your azconfig code easier?

Reply:
That makes AzConfig to have more public APIs to maintain. 🔢